src/main/scala/com/danielasfregola/twitter4s/http/serializers/CustomFormats.scala

```diff
@@ -18,9 +18,9 @@ private[twitter4s] object CustomFormats extends FormatsComposer {
 private[twitter4s] case object InstantSerializer
     extends CustomSerializer[Instant](format =>
       ({
-        case JInt(i)                                            => DateTimeFormatter.parseInstant(i.toLong)
-        case JLong(l)                                           => DateTimeFormatter.parseInstant(l)
-        case JString(s) if DateTimeFormatter.canParseInstant(s) => DateTimeFormatter.parseInstant(s)
+        case JInt(i)                                                         => DateTimeFormatter.parseInstant(i.toLong)
+        case JLong(l)                                                        => DateTimeFormatter.parseInstant(l)
+        case JString(s) if DateTimeFormatter.parseInstant(s).isDefined       => DateTimeFormatter.parseInstant(s).get
```

Comment:
Not a fan of `Option.get` ...let me see if we can keep the original signature!

Reply:
Yes, I believe you can avoid the `parseInstant(s).isDefined` and  `parseInstant(s).get` here if you keep the same signature as suggested in the comment